hbase_storage/src/main/java/com/splicemachine/si/data/hbase/coprocessor/SIObserver.java

```diff
@@ -230,7 +229,7 @@ public InternalScanner preCompact(ObserverContext<RegionCoprocessorEnvironment>
                 }
                 SICompactionScanner siScanner = new SICompactionScanner(
                         state, scanner, purgeConfig,
-                        conf.getOlapCompactionResolutionShare(), conf.getOlapCompactionResolutionBufferSize(), context);
+                        conf.getOlapCompactionResolutionShare(), conf.getLocalCompactionResolutionBufferSize(), context);
                 siScanner.start();
```

Comment:
Why the buffer size is set to LocalCompactionResolutionBufferSize for precompact()? Should it be used for preFlush() instead?

Reply:
This affects compactions that couldn't be run in Spark and fallback to run in HBase. I think flushes are fine because we are dealing with less data and the data itself is already in the MemStore, we are not bringing new data from a file into memory, so having a larger buffer shouldn't be a problem.